java/configuration/src/main/java/sleeper/configuration/properties/instance/GarbageCollectionProperty.java

```diff
@@ -27,11 +27,18 @@ public interface GarbageCollectionProperty {
             .defaultValue("15")
             .propertyGroup(InstancePropertyGroup.GARBAGE_COLLECTOR)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty GARBAGE_COLLECTOR_LAMBDA_TIMEOUT_IN_MINUTES = Index.propertyBuilder("sleeper.gc.lambda.timeout.minutes")
+            .description("The configurable timeout wait (minutes) for the garbage collector lambda. When not set defaults to the garbage collector period.")
+            .defaultValue("15")
+            .propertyGroup(InstancePropertyGroup.GARBAGE_COLLECTOR)
+            .validationPredicate(Utils::isPositiveIntegerLtEq900)
```

Comment:
It looks like this maximum is based on seconds rather than minutes.

Reply:
Predicate correctly updated